docs/best-practices/data-partitioning-strategies.md

```diff
@@ -117,9 +117,9 @@ Consider the following points when you design your entities for Azure table stor
 
 For more information, see [Azure storage table design guide] and [Scalable partitioning strategy].
 
-## Partitioning Azure blob storage
+## Partitioning Azure Blob Storage
 
-Azure blob storage makes it possible to hold large binary objects. Use block blobs in scenarios when you need to upload or download large volumes of data quickly. Use page blobs for applications that require random rather than serial access to parts of the data.
+Azure Blob Storage makes it possible to hold large binary objects. Use block blobs in scenarios when you need to upload or download large volumes of data quickly. Use page blobs for applications that require random rather than serial access to parts of the data.
```

Comment:
It's actually "Azure Blob storage" (lower s in storage). Any chance you can update this PR?  Thanks!